docs/source/api/entities.md

```diff
@@ -0,0 +1,329 @@
+      "status": "ACTIVE",
+      "published": "2020-01-01T00:00Z",
+      "commerce": {
+        "salesUnit": "ST",
+        "salesQuantityPerUnit": "1",
+        "pricing": [
+          {
+            "isTaxable": true,
+            "isNetPrice": true,
+            "countryCode": "CH",
+            "currencyCode": "CHF",
+            "amount": 10000,
+            "maxQuantity": null
+          }
+        ]
+      },
+      "warehousing": {
+        "baseUnit": "ST",
+        "sku": ""
+      },
+      "supply": {
+        "weightInGram": 0,
+        "heightInMillimeters": 0,
+        "lengthInMillimeters": 0,
+        "widthInMillimeters": 0
+      },
+      "proxy": {
+        "assignments": [
+          {
+            "vector": {
+              "color": "red"
+            },
+            "productId": "B"
+          }
+        ]
+      },
+      "plan": {
+        "billingInterval": "daily",
+        "billingIntervalCount": 1,
+        "usageCalculationType": "metered",
+        "trialInterval": "daily",
+        "trialIntervalCount": 1
+      },
+      "bundleItems": [
+        {
+          "productId": "c",
+          "quantity": 1,
+          "configuration": [
+            {
+              "key": "greeting",
+              "value": "For my Darling"
+            }
+          ]
+        }
+      ],
+      "meta": {},
+      "content": {
+        "de": {
+          "created": null,
+          "updated": null,
+          "vendor": "Herstellername",
+          "brand": "Marke",
+          "title": "Produktname",
+          "slug": "produktname",
+          "subtitle": "Short description",
+          "description": "Long description",
+          "labels": [
+            "Neu"
+          ]
+        }
+      }
+    },
+    "media": [
+      {
+        "_id": null,
+        "created": null,
+        "updated": null,
+        "asset": {
+          "_id": null,
+          "contentHash": null,
+          "url": "https://www.story.one/media/images/poop-4108423_1920.width-1600.format-jpeg.jpg"
+        },
+        "tags": ["big"],
+        "meta": {},
+        "content": {
+          "de": {
+            "created": null,
+            "updated": null,
+            "title": "Produktname",
+            "subtitle": "Short description",
+          }
+        }
+      }
+    ],
+    "variations": [
+      {
+        "_id": null,
+        "created": null,
+        "updated": null,
+        "key": "color",
+        "type": "COLOR",
+        "options": [
+          {
+            "value": "ff0000",
+            "content": {
+              "de": {
+                "created": null,
+                "updated": null,
+                "title": "Rot",
+                "subtitle": ""
+              }
+            }
+          }
+        ],
+        "content": {
+          "de": {
+            "created": null,
+            "updated": null,
+            "title": "Farbe",
+            "subtitle": "Farbvariante"
+          }
+        }
+      }
+    ],
+    "reviews": [
```

Comment:
Reviews should probably be in a separate entity because there will be a lot of appending operations (e.g. add a review to a product) which could probably slow down the import if they are all updating the product itself.